pkg/controller/masteruserrecord/sync.go

```diff
@@ -205,14 +207,33 @@ func (s *Synchronizer) openShift3XConsoleURL(apiEndpoint string) (string, error)
 	return url, nil
 }
 
+func (s *Synchronizer) alignDisabled() {
+	for _, uaStatus := range s.record.Status.UserAccounts {
+		if !isDisabled(uaStatus.Conditions) {
+			return
+		}
+	}
+	s.record.Status.Conditions, _ = condition.AddOrUpdateStatusConditions(s.record.Status.Conditions, toBeDisabled())
```

Comment:
Just to be on the safe side in case there is no embedded UserAccounts.
```suggestion
	if len(s.record.Status.Conditions) > 0 {
		s.record.Status.Conditions, _ = condition.AddOrUpdateStatusConditions(s.record.Status.Conditions, toBeDisabled())
	}
```